Source/Core/DolphinQt/Config/Mapping/MappingWindow.cpp

```diff
@@ -348,7 +349,8 @@ void MappingWindow::SetMappingType(MappingWindow::Type type)
     widget = new WiimoteEmuGeneral(this, extension);
     setWindowTitle(tr("Wii Remote %1").arg(GetPort() + 1));
     AddWidget(tr("General and Options"), widget);
-    AddWidget(tr("Motion Controls"), new WiimoteEmuMotionControl(this));
+    AddWidget(tr("Motion Controls (Emulated)"), new WiimoteEmuMotionControl(this));
+    AddWidget(tr("Motion Controls (Real)"), new WiimoteEmuMotionControl2(this));
```

Comment:
Not sure I'm too happy about the "...2" Naming. Thoughts on using "...Real", "...Controller" or...I'd probably even be ok with "...IMU" at this point...instead of the "...2" Suffix? Or any other suffix that makes sense to distnguish it from the other one.

Reply:
Yes, I was unsure about this myself. I think "...IMU" would be more consistent with the other classes I have added. Should I leave the name of the original (`WiimoteEmuMotionControl`) as is?

Reply:
If it helps further distinguishing the two, yes. But I don't think it is really necessary at this point.
Then again; I haven't looked at the UI so I might be wrong.